content/en/docs/tutorials/security/apparmor.md

```diff
@@ -152,7 +132,7 @@ To verify that the profile was applied, you can look for the AppArmor security o
 kubectl get events | grep Created
```

Comment:
I think this could (should) use a field selector. I haven't checked that that field is supported though.

Also, I recommend using the `events.k8s.io/v1` API group.

Reply:
Added field selector on pod name in [91e1830](https://github.com/kubernetes/website/pull/37191/commits/91e1830d8d414285e5d5e05b94de3f5f4f4fc94f)